lxd/cluster/heartbeat.go

```diff
@@ -35,8 +36,9 @@ type APIHeartbeatVersion struct {
 
 // APIHeartbeat contains data sent to nodes in heartbeat.
 type APIHeartbeat struct {
-	sync.Mutex // Used to control access to Members maps.
-	Members    map[int64]APIHeartbeatMember
+	sync.Mutex                              // Used to control access to Members maps.
+	Members    map[int64]APIHeartbeatMember // Deprecated, use Nodes instead.
+	Nodes      map[string]APIHeartbeatMember
```

Comment:
@stgraber @freeekanayaka by adding a new field called Nodes to this struct, it will ensure that if an older node (whose struct doesnt have the Nodes field) receives a heartbeat from a new node containing the Nodes field, then it will just ignore it. We still send the existing (albeit broken) Members field for now too.